js/msp/MSPCodes.js

```diff
@@ -76,9 +76,8 @@ var MSPCodes = {
     MSP_SET_PID_ADVANCED:           95,
     MSP_SENSOR_CONFIG:              96,
     MSP_SET_SENSOR_CONFIG:          97,
-    //MSP_SPECIAL_PARAMETERS:         98, // DEPRECATED
-    //MSP_SET_SPECIAL_PARAMETERS:     99, // DEPRECATED
-    //MSP_IDENT:                      100, // DEPRECTED
+    MSP_CAMERA_CONTROL:             98,
```

Comment:
We shouldn't reuse MSP codes. Can you please point to the PR that introduced `MSP_CAMERA_CONTROL` in firmware, so we can ask the author to change this to a new code, and then can you please reinstate the commented out lines to make this problem less likely to happen again in the future?